tests/manage/test_run_io_multiple_pods.py

```diff
@@ -1,19 +1,69 @@
 import pytest
 import logging
 import random
+from tests import helpers
 from ocs_ci.ocs import constants
 
 from ocs_ci.framework.testlib import ManageTest, tier2
-from tests.fixtures import (
-    create_rbd_storageclass, create_ceph_block_pool,
-    create_rbd_secret, create_pods, create_pvcs, create_project,
-    create_cephfs_secret, create_cephfs_storageclass
-)
 
 
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture()
+def create_pvcs(request):
+    """
+    Create multiple PVCs
+    """
+    class_instance = request.node.cls
+
+    def finalizer():
+        """
+        Delete multiple PVCs
+        """
+        if hasattr(class_instance, 'pvc_objs'):
+            for pvc_obj in class_instance.pvc_objs:
+                pvc_obj.delete()
+            for pvc_obj in class_instance.pvc_objs:
+                pvc_obj.ocp.wait_for_delete(pvc_obj.name)
+
+    request.addfinalizer(finalizer)
+
+    class_instance.pvc_objs = helpers.create_multiple_pvcs(
+        sc_name=class_instance.sc_obj.name, number_of_pvc=class_instance.num_of_pvcs,
+        size=class_instance.pvc_size, namespace=class_instance.namespace
+    )
+
+
+@pytest.fixture()
+def create_pods(request):
+    """
+    Create multiple pods
+    """
+    class_instance = request.node.cls
+
+    def finalizer():
+        """
+        Delete multiple pods
+        """
+        if hasattr(class_instance, 'pod_objs'):
```

Comment:
Same comment as above. Let's not depend on class attributes to store created pod objects. We can just register finalizers directly here for each created pod.